librarywidget.py

```diff
@@ -525,21 +524,22 @@ def foldersWidget(self):
         """
         return self._foldersWidget
 
-    def folderIgnoreFilter(self):
+    def folderIgnoreFilter(self, path):
         """
-        Return a list of folder names that should be hidden/ignored.
+        Return True if path have to be ignored
 
-        :rtype: list[str]
+        :rtype: bool
         """
         ignoreFilter = ['.', '.studiolibrary', ".mayaswatches"]
 
         if not self.isTrashFolderVisible():
             ignoreFilter.append('trash')
 
-        for ext in studiolibrary.itemExtensions():
-            ignoreFilter.append(ext)
+        for f in ignoreFilter + studiolibrary.itemExtensions():
```

Comment:
I like that you have used the "+" to concatenate the list! Do you mind moving this to before the condition?

```python
        ignoreNames = ['.', '.studiolibrary', ".mayaswatches"]
        ignoreNames += studiolibrary.itemExtensions()

        ...

        for ignoreName in ignoreNames:
            ...